Modules/Promotion/Http/Requests/StorePromotionRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Modules\Promotion\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StorePromotionRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
```

Comment:
Promotion contains a lot of fields but we are validating only title, all other fields should pass according validation as well